UKHomeOffice/nmsw-ui
# NMSW-24 Setup analytics tracking for NMSW

# Ticket

NMSW-24

----

## AC



----

## To test

- Get DevOps to add you to the GA account for dev
- open GA in a different browser
- open the live tracking
- open nmsw app and set cookies to accepted
- > you should soon see a count in the live tracking
- open nmsw in another browser
- leave cookies unselected
- > you should see the count remaining at 1
- select accepted
- > you should see the count go to 2
- change to declined
- > you should see the count return to 1

----

## Notes

- You may need to keep going back to first browser and clicking on things so GA knows you're still active on the page
